lecture_notes/08 collections/1-array.js

```diff
@@ -1,8 +1,50 @@
+// PUSH
 'use strict';
 
 const ages = [10, 12, 15, 15, 17, 18, 18, 19, 20];
+const total = ages.push(21, 21, 22);
 
-const first = ages[0];
-const last = ages[ages.length - 1];
+console.log(ages); // [10, 12, 15, 15, 17, 18, 18, 19, 20, 21, 21, 22]
+
+
+// POP
+'use strict';
```

Comment:
`'use strict';` should be always at the beginning of the file or beginning of the function